src/bamber19_icesheets/cli.py

```diff
@@ -0,0 +1,242 @@
+from bamber19_icesheets.bamber19_icesheets_preprocess import (
+    bamber19_preprocess_icesheets,
+)
+from bamber19_icesheets.bamber19_icesheets_project import (
+    bamber19_project_icesheets,
+    bamber19_project_icesheets_temperaturedriven,
+)
+from bamber19_icesheets.bamber19_icesheets_postprocess import (
+    bamber19_postprocess_icesheets,
+)
+
+import click
+
+
+@click.command()
+@click.option(
+    "--pyear-start",
+    envvar="BAMBER19_ICESHEETS_PYEAR_START",
+    help="Projection year start [default=2020]",
+    default=2020,
+    type=click.IntRange(min=2020),
+)
+@click.option(
+    "--pyear-end",
+    envvar="BAMBER19_ICESHEETS_PYEAR_END",
+    help="Projection year end [default=2100]",
+    default=2100,
+    type=click.IntRange(max=2300),
+)
+@click.option(
+    "--pyear-step",
+    envvar="BAMBER19_ICESHEETS_PYEAR_STEP",
+    help="Projection year step [default=10]",
+    default=10,
+    type=click.IntRange(min=1),
+)
+@click.option(
+    "--baseyear",
+    envvar="BAMBER19_ICESHEETS_BASEYEAR",
+    help="Year to which projections are referenced [default = 2000]",
+    default=2000,
+)
+@click.option(
+    "--scenario",
+    envvar="BAMBER19_ICESHEETS_SCENARIO",
+    help="Emissions scenario of interest [default=ssp585]",
+    default="ssp585",
```

Comment:
Stop me if I'm not following the issue here and in the comment below. But, https://github.com/fact-sealevel/tlm-sterodynamics/issues/7 was about me screwing up the original module's default. It was fixing a typo and not about having an inconsistent default scenario between modules.

If the original module had `"rcp85"` then we want to stick with that for the initial release, just to be faithful to the original.

```suggestion
    default="rcp85",
```
Does that sound right? Should raise this with @kemccusker as it could be a wider issue to deal with later. @kemccusker, do you have other thoughts on this?

Reply:
oh! I misunderstood that, keeping rcp85 simplifies all of that, that sounds great to me. i'll take out the stuff in `cli.py` checking for the incompatible scenario etc.

Reply:
@brews sorry I missed this (now resolved convo and merged PR). I think this is the right call, to stick with the original discussion. There IS a wider discussion to be had about defaults across modules but I think that's to be handled at a higher level ("experiment" probably).